oleesir/authors-haven
# Login user

**What does this Merge request do?**

Implements feat__loginUser

**Description of Tasks completed?**

Implements feat__feat__loginUser

**How should this be manually tested?**

- git fetch on your local machine and check out to feat__loginUser branch
- Run "yarn start" to start the development server

**Any background context you want to provide?**

N/A


**Screenshots (if appropriate)**

<img width="1422" alt="Screenshot 2021-11-24 at 1 18 17 PM" src="https://user-images.githubusercontent.com/17679511/143237552-8cc52dc2-4d99-4905-a209-f469e0fd0495.png">
<img width="1422" alt="Screenshot 2021-11-24 at 1 18 58 PM" src="https://user-images.githubusercontent.com/17679511/143237561-0c4850df-7677-495d-882a-051e143f681a.png">
